client/src/context/i18n/i18n.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC, Fragment, ReactNode } from 'react';
+import { IntlProvider } from 'react-intl';
+import flatten from 'flat';
+import messages from '../../messages';
+
+import { LOCALES } from '../appContext/const';
+
+interface Props {
+  children: ReactNode,
+  locale: string,
+}
+
+export const I18Provider: FC<Props> = (props) => {
+  const { children, locale = LOCALES.ENGLISH } = props;
+
+  return (
+    <IntlProvider
+      textComponent={Fragment}
+      locale={locale}
+      messages={flatten(messages[locale])}
```

Comment:
move to the variable
```suggestion
      messages={message}
```